content/docs/central-dash/registration-flow.md

```diff
@@ -0,0 +1,25 @@
++++
+title = "Registration Flow"
+description = "The view that allows you to set up your first namespace in the cluster"
+weight = 10
++++
+
+Your Kubeflow deployment comes deployed with Kubeflow Central Dashboard. 
+
+This view will show when the cluster is:
+- When in `multi-user` isolation mode:
```

Comment:
This list doesn't render. See preview: https://deploy-preview-1538--competent-brattain-de2d6d.netlify.com/docs/central-dash/registration-flow/